### PR TITLE
Fixes pdf.combined.js for webpack

### LINKED ISSUE #8057: pdfjs-dist pdf.combined.js build is broken
It appears the commit below removed all reference to the variable pdfjslibs. I'm assuming it is being used somewhere else in the combined build.

Link to PDF file (or attach file here):
https://github.com/mozilla/pdf.js/pull/8046

Configuration:
- Web browser and its version: Chrome 56.0.2924.87
- Operating system and its version: Mac osx
- PDF.js version: pdfjs-dist 1.7.274
- Is an extension:

Steps to reproduce the problem:
1. attempt to use the pdfjs-dist combined package

What is the expected behavior? (add screenshot)
PDFJS should be found as a variable on the window

What went wrong? (add screenshot)
screen shot 2017-02-10 at 11 28 12 am

### PR DESCRIPTION
**Pull request by yurydelendik**
Fixes #8057

### CODE REVIEW

**Review by Snuffleupagus — APPROVED** [review]
Looks good, thank you!

File: src/display/api.js [hunk 1]
```diff
@@ -12,7 +12,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-/* globals requirejs, pdfjsLibs, __webpack_require__, __pdfjsdev_webpack__ */
+/* globals requirejs, __webpack_require__, __pdfjsdev_webpack__ */
```
Comment (Snuffleupagus) [inline src/display/api.js]: Nit: `__webpack_require__` looks unused, can we remove that one as well?

**Comment by MSchip** [comment]
Thanks for taking a look at this so quickly - much appreciated. Does the pdfjs-dist build automatically following a successful merge?

**Comment by yurydelendik** [comment]
> Does the pdfjs-dist build automatically following a successful merge?

yes

**Pull request merged**